e2e/simple_ondatra_test.go

```diff
@@ -185,12 +185,15 @@ func TestMirror(t *testing.T) {
 	stopTwoPortMirror(t, client)
 }
 
-func addMPLSFlow(t *testing.T, otgCfg gosnappi.Config, name, srcName, dstName, srcv4, dstv4 string) {
+func addMPLSFlow(t *testing.T, otgCfg gosnappi.Config, name, srcName, dstName, srcv4, dstv4 string, pps uint64, totPackets uint32) {
```

Comment:
perhaps let's document the behavior here explicitly, as these are coupled in a way

if totPackets is 0, we continue to send pps packets per second indefinitely until stopped, right? and if totPackets is non-zero, we send pps packets per second until totPackets # of packets are sent?

Reply:
Done.